lib/Runtime/Language/JavascriptExceptionOperators.cpp

```diff
@@ -440,6 +466,13 @@ namespace Js
         catch(const Js::JavascriptException& err)
         {
             pExceptionObject = err.GetAndClear();
+#if ENABLE_NATIVE_CODEGEN
+            if (scriptContext->GetThreadContext()->GetTryCatchFrameAddr() != nullptr)
+            {
+                Assert(pExceptionObject->GetExceptionContext() && pExceptionObject->GetExceptionContext()->ThrowingFunction());
+                WalkStackForCleaningUpInlineeInfo(scriptContext, pExceptionObject->GetExceptionContext()->ThrowingFunction());
```

Comment:
>pExceptionObject->GetExceptionContext()->ThrowingFunction() [](start = 65, length = 59)

Clean this up as this parameter is not really used and is the wrong thing to pass here anyway